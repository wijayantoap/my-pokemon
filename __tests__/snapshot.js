import React from "react";
import renderer from "react-test-renderer";
import Details from "../pages/details/[name]";
import List from "../pages/list";
import Index from "../pages/index";

it("renders homepage unchanged", () => {
  const tree = renderer.create(<Index />).toJSON();
  expect(tree).toMatchSnapshot();
});

it("renders details unchanged", () => {
  const tree = renderer.create(<Details />).toJSON();
  expect(tree).toMatchSnapshot();
});

it("renders list unchanged", () => {
  const tree = renderer.create(<List />).toJSON();
  expect(tree).toMatchSnapshot();
});
