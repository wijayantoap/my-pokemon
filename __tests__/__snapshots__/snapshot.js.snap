// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders details unchanged 1`] = `
<p>
  Something went wrong
</p>
`;

exports[`renders homepage unchanged 1`] = `
<p>
  Something went wrong
</p>
`;

exports[`renders list unchanged 1`] = `
<div
  className="css-9hae16"
>
  <div
    className="headroom-wrapper"
    style={
      Object {
        "height": null,
      }
    }
  >
    <div
      className="headroom headroom--unfixed"
      style={
        Object {
          "MsTransform": "translate3D(0, 0, 0)",
          "WebkitTransform": "translate3D(0, 0, 0)",
          "left": 0,
          "position": "relative",
          "right": 0,
          "top": 0,
          "transform": "translate3D(0, 0, 0)",
          "zIndex": 1,
        }
      }
    >
      <header
        className="css-1jqswcl"
      >
        <div>
          <h2
            className="css-p9qu6"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            Home
          </h2>
        </div>
      </header>
    </div>
  </div>
  <div
    className="css-1rsklg3"
  />
  <div
    className="css-1oskuwq"
  >
    <h1
      style={
        Object {
          "textAlign": "center",
        }
      }
    >
      Try catching your first Pok√©mon
    </h1>
  </div>
</div>
`;
